conan/tools/apple/xcodedeps.py

```diff
@@ -239,6 +255,11 @@ def get_content_for_component(self, pkg_name, component_name, cpp_info, reqs):
         result[file_dep_name] = dep_content
         return result
 
+    def _add_component(self, pkg_name, component_name):
```

Comment:
I think the function name sounds weird because we're not adding any component at all... Could we rename it?